README.md

```diff
@@ -190,6 +190,11 @@ for a type named something else like `q`, it will be `show_q`.
 #### Debug Mode
 If Hazel is hanging on load or when you perform certain actions, you can load into Debug Mode by appending `#debug` to the URL and reloading. From there, you have some buttons that will change settings or reset local storage. Refresh without the `#debug` flag and hopefully you can resolve the situation from there.
 
+#### Testing
+You can run all of the unit tests located in `src/haz3ltest` by running `make test`.
+
+Unit tests are written using the [Tezt framework](https://gitlab.com/nomadic-labs/tezt/).
+
```

Comment:
We should also mention `make reset-regression-tests` and `make fix-test-answers` (or remove them from the Makefile).